great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -487,6 +489,10 @@ def _save_expectation_suite(
         return self.expectations_store.set(key, expectation_suite, **kwargs)
 
     # Properties
+    @property
+    def anonymizer(self) -> Anonymizer:
+        return self._anonymizer
```

Comment:
Don't love adding more properties but if the anonymizer relies directly on the `data_context_id` for hashing, we need to make it publicly available